spec.md

```diff
@@ -1450,6 +1464,45 @@ message NodeServiceCapability {
 
 If the plugin is unable to complete the NodeGetCapabilities call successfully, it MUST return a non-ok gRPC code in the gRPC status.
 
+
+#### `NodeGetInfo`
+
+A Node Plugin MUST implement this RPC call if the plugin has `PUBLISH_UNPUBLISH_VOLUME` controller capability.
+The Plugin SHALL assume that this RPC will be executed on the node where the volume will be used.
+The CO SHOULD call this RPC for the node at which it wants to place the workload.
+The result of this call will be used by CO in `ControllerPublishVolume`.
+
+
+```protobuf
+message NodeGetInfoRequest {
+}
+
+message NodeGetInfoResponse {
+  // The ID of the node as understood by the SP which SHALL be used by
+  // CO in subsequent `ControllerPublishVolume`.
+  // This is a REQUIRED field.
+  string node_id = 1;
+
+  // Maximum number of volumes that controller can publish
+  // to the node. If value is not set or zero CO SHALL
+  // decide how many volumes of this type can be published
+  // by the controller to the node.
+  // This field is OPTIONAL
```

Comment:
nit: please end w/ a period.

Reply:
also, maybe include a statement along the lines that negative values are not allowed here